backbone/src/main/java/org/researchstack/backbone/utils/ResUtils.java

```diff
@@ -13,6 +13,12 @@
 
 public class ResUtils
 {
+    public static final String LOGO_DISEASE = "logo_disease";
```

Comment:
Well, this resource is a little different than the patterns of the rest.  Most of the resources in backbone have a default value, which in this case would be a default image, that shows up if the app doesn't override that resource name.  However, this one just assumes all apps will override it with their own and it will be named "logo_disease".  So, that is why it doesn't have "rsb_" and the compiler won't complain.